screens/ResourceScreen.js

```diff
@@ -11,12 +11,13 @@ const ResourceScreen = () => {
   return (
     <Container>
     <Header leftButton="menu" onLeftButtonPress={() => Actions.drawerOpen()}>Resources</Header>
-
     <Content>
-      <View style={styles.container}>
-        <Text style={styles.welcome}>
-          Resource Screen
-        </Text>
+      <View>
+        <Button variant = "secondary" style = {styles.container}>naloxone injection instructions</Button>
+        <Button variant = "secondary" style = {styles.container}>naloxone nasal spray instructions</Button>
+        <Button variant = "secondary" style = {styles.container}>call 911</Button>
+        <Button variant = "secondary"style = {styles.container}>your rights</Button>
+        <Button variant = "primary" style = {styles.container}>exit</Button>
```

Comment:
Redirect to the main screen like this
```suggestion
        <Button variant = "primary" style = {styles.container} onPress={() => Actions.main()}>exit</Button>
```